manifests/g/GovernmentOfThePRC/GuangdongProvince/YueZhengYi/2.5.20000/GovernmentOfThePRC.GuangdongProvince.YueZhengYi.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: GovernmentOfThePRC.GuangdongProvince.YueZhengYi
+PackageVersion: 2.5.20000
```

Comment:
The version number on the website appears to be incorrect or out of date...

Reply:
Yeah, the version number on the website seems to be two or three years old...